go-controller/pkg/ovn/ovn.go

```diff
@@ -122,6 +122,9 @@ const (
 	SCTP = "SCTP"
 )
 
+// Used for the syncPeriodic go routine
+var startSyncPeriodicOnce sync.Once
+
```

Comment:
Why do you need this? The function Run() is called only once, so you are good without it, correct?

Reply:
it's just a pattern i have picked to make sure there is a single go-routine for syncPeriodic.

Reply:
If the go routine is being called from within a function that itself will be called several times, then it makes sense to use sync.Once. However, in this case we already know that it Run() is going to be called only once, so there is no need for sync.Once, IMHO.

Reply:
@abhat the ovn-master leader election is separate from the RAFT.

When an ovnkube-master stops leading, we just exit and re-start the process. When this process comes back up, it becomes a follower with clean state and cache. So, from the code inspection I don't see Run() being called twice. That said, I am fine if you want to keep it.